### PR TITLE
6.1 experience

### PR DESCRIPTION
**Pull request by bwrsandman**
Port of https://code.launchpad.net/~savoirfairelinux-openerp/openerp-hr/experience/+merge/194926

### CODE REVIEW

File: hr_experience/hr_academic.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        'employee_id': fields.many2one('hr.employee', 'Employee', required=True),
+        'start_date': fields.date('Start date'),
+        'end_date': fields.date('End date'),
+        'description': fields.text('Description', translate=True),
+        'partner_id': fields.many2one('res.partner', 'School or University'),
+        'location': fields.char('Location', size=64, translate=True),
+        'diploma': fields.char('Diploma', size=64, translate=True),
+        'study_field': fields.char('Field of study', size=64, translate=True),
+        'activities': fields.text('Activities and associations', translate=True),
+        'expire': fields.boolean('Expire'),
```
Comment (ghost) [inline hr_experience/hr_academic.py:38]: Is this field (expire) really necessary? Isn't end_date enough?

Comment (max3903) [inline hr_experience/hr_academic.py:38]: The date end field is displayed if the expire is true.

Comment (ghost) [inline hr_experience/hr_academic.py:38]: Makes sense!

**Comment by ghost** [comment]
:+1:

**Comment by dreispt** [comment]
:+1:

**Pull request merged**